web/src/components/admin/settings/forges/AdminForgeForm.vue

```diff
@@ -19,36 +19,56 @@
       />
     </InputField>
 
-    <InputField v-slot="{ id }" :label="$t('url')">
+    <InputField v-if="forge.type !== 'bitbucket'" v-slot="{ id }" :label="$t('url')">
       <TextField :id="id" v-model="forge.url" required />
     </InputField>
 
     <template v-if="forge.type && forge.url">
       <hr class="my-4 border-gray-800" />
 
       <InputField v-slot="{ id }" :label="$t('oauth_redirect_url')">
-        <i18n-t keypath="use_this_redirect_uri_to_create" tag="p" class="mb-2">
+        {{ $t('use_this_redirect_uri_to_create') }}
+        <i18n-t v-if="forge.type !== 'addon'" keypath="developer_settings_to_create" tag="p">
```

Comment:
Same here, I know not introduced by you, but this should use an InputField label.

Reply:
I would do so. However we dont need to address this in this PR.